src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationSubmittedNotification.java

```diff
@@ -28,8 +28,7 @@
     public void send(CaseData caseData, Long id) {
         Map<String, String> templateVars = commonContent.templateVarsFor(caseData);
 
-        caseData.setDueDate(caseData.getApplication().getDateOfSubmissionResponse());
-        templateVars.put(SUBMISSION_RESPONSE_DATE, caseData.getDueDate().format(dateTimeFormatter));
+        templateVars.put(SUBMISSION_RESPONSE_DATE, caseData.getApplication().getDateOfSubmissionResponse().format(dateTimeFormatter));
```

Comment:
this should just use due date, but not set it (it should have already been set as part of submission)